<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationWindow</class>
 <widget class="QMainWindow" name="SimulationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cross the Bridge</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numOfOld">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>129</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="numOfMiddleAged">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>129</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="numOfYoung">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>129</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="numOfYoungLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>129</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Young (1min)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="numOfMiddleAgedLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>129</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Middle-aged (2min)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="numOfOldLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>129</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Old (3min)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="leftBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>151</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>Left side of bridge</string>
    </property>
   </widget>
   <widget class="QLabel" name="rightBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>151</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>Right side of bridge</string>
    </property>
   </widget>
   <widget class="QLabel" name="bridgeBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>101</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>the bridge</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeTakenLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time taken: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
